lib/Appium/TouchActions.pm

```diff
@@ -6,7 +6,7 @@ use Moo;
 has 'driver' => (
     is => 'ro',
     required => 1,
-    handles => [ qw/execute_script/ ]
+    handles => [ qw/prompt/ ]
```

Comment:
it should be

*has 'driver' => (*
*    is => 'ro',*
*    required => 1,*
*    handles => [ qw/perform execute_script/ ]*
*);*


*if you want i will fix this.*

On Thu, Sep 8, 2016 at 8:26 PM, Daniel Gempesaw <notifications@github.com>
wrote:

> In lib/Appium/TouchActions.pm
> <https://github.com/appium/perl-client/pull/13#discussion_r78023110>:
>
> > @@ -6,7 +6,7 @@ use Moo;
> >  has 'driver' => (
> >      is => 'ro',
> >      required => 1,
> > -    handles => [ qw/execute_script/ ]
> > +    handles => [ qw/prompt/ ]
>
> Sorry, I don't understand what this handles entry is for. I searched for
> prompt in TouchActions.pm and Appium.pm but didn't find any entries. Do
> you remember why it's here?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/appium/perl-client/pull/13/files/35beb40d930706e5686fcc9df8b6dc6ce5917da1#r78023110>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AINJQZi0hDEAby7bkmSXe7NFB8IEBBJGks5qoCI6gaJpZM4Jhz9t>
> .
>